src/pages/_app.tsx

```diff
@@ -89,5 +103,21 @@ class ErrorBoundary extends React.Component {
   }
 }
 
+const Center: React.FC = ({ children }) => {
+  return (
+    <div
+      style={{
+        width: '100vw',
+        height: '100vh',
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}
```

Comment:
Minor comment here. This was probably left in the Liikunta app in accident when the solution was being PoC. In that sense I don't recommend using a style prop here, but I'm also perfectly fine with it as it only wraps the loading spinner.

Reply:
I understand your concern but I'm also fine with this style props in this case.